docs/plugin/interfaces/tags.md

```diff
@@ -0,0 +1,483 @@
+When designing a COM-RPC interface, it is possible to add specific comments (known as annotations or tags) that will drive the code generation. 
+
+All tags follow the same format of `@` followed by the tag name. They can be inserted into the interface using either inline `//` comments or `/* */` block comments. Tags can influence the generation of the COM-RPC ProxyStubs and the generated JSON-RPC interfaces. They can also be used to configure the documentation generation
+
+## Summary
+
+### General purpose tags
+
+| Tag|Short Description|Deprecated|StubGen|JsonGen|Scope|
+|--|--|:--:|:--:|:--:|:--:|
+|[@stubgen:skip](#stubgenskip)|Stop processing current file, now deprecated| Yes | Yes|Yes|File|
+|[@stop](#stop)|Same as @stubgen:skip, not deprecated, but also never advertised ||Yes| Yes|File|
```

Comment:
That is an odd thing to say in a documentation (not advertised). We wish to get rid of `@stubgen:skip` in favour of `@omit`, hence the deprecation, but `@stop` is still there as a last resort when the generator is technically unable to parse the file, though in such case it is recommended that the complicated part is moved to a separate header file instead.

Reply:
Was copy/pasted from the old copy of the documentation. Will clarify